eng/packages/http-client-csharp/generator/TestProjects/Local/Basic-TypeSpec/src/Generated/BasicTypeSpecClient.RestClient.cs

```diff
@@ -14,21 +14,21 @@ namespace BasicTypeSpec
     /// <summary></summary>
     public partial class BasicTypeSpecClient
     {
-        private static PipelineMessageClassifier _pipelineMessageClassifier200;
-        private static PipelineMessageClassifier _pipelineMessageClassifier204;
+        private static System.ClientModel.Primitives.PipelineMessageClassifier _pipelineMessageClassifier200;
```

Comment:
Why do such change occur？

Reply:
It was caused by the change in MGC, updated now.